de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/events/PublishToolEvent.java

```diff
@@ -0,0 +1,47 @@
+package org.iplantc.de.admin.desktop.client.toolAdmin.events;
+
+import org.iplantc.de.client.models.tool.Tool;
+
+import com.google.gwt.event.shared.EventHandler;
+import com.google.gwt.event.shared.GwtEvent;
+import com.google.gwt.event.shared.HandlerRegistration;
+
+/**
+ * Created by sriram on 6/1/17.
+ */
+public class PublishToolEvent extends GwtEvent<PublishToolEvent.PublishToolEventHandler> {
+
+    public static Type<PublishToolEventHandler> TYPE = new Type<>();
+
+    private Tool tool;
+
+    public PublishToolEvent(Tool tool) {
+        this.tool = tool;
+    }
+
+    public Tool getTool() {
+        return tool;
+    }
+
+    public void setTool(Tool tool) {
```

Comment:
Is setTool() used?  I think the tool passed into the constructor is all that is used.